### PR TITLE
fix: update models to match helpdesk v3.3.0

### PR DESCRIPTION
**Pull request by Gaitholabi**

### CODE REVIEW

**Review by bytestream — APPROVED** [review]

File: test/Unit/Cache/CacheStrategyConfiguratorTest.php [hunk 1]
```diff
@@ -36,7 +36,10 @@ protected function setUp(): void
 
     public function testConfigureCache(): void
     {
-        $this->apiCacheMap->getCacheableApis('/api/')->shouldBeCalled();
+        $apiCacheMap = new ApiCacheMap;
+        $this->apiCacheMap
+            ->getCacheableApis('/api/')->shouldBeCalled()
+            ->willReturn($apiCacheMap->getCacheableApis('/api/'));
```
Comment (jshah4517) [inline test/Unit/Cache/CacheStrategyConfiguratorTest.php:42]: Curious what the reason for this change was.

File: src/Model/Ticket/SlaPlan.php [hunk 2]
```diff
@@ -206,4 +212,23 @@ public function setName(string $name): self
...
+    public function getTranslations(): ?array
+    {
+        return $this->translations;
+    }
+
+    /**
+     * @param SlaPlanTranslation[]|null $translations
+     * @return self
+     */
+    public function setTranslations(?array $translations): self
```
Comment (jshah4517) [inline src/Model/Ticket/SlaPlan.php:228]: Are the types for this and the get function correct?

File: test/Unit/Cache/CacheStrategyConfiguratorTest.php (hunk 1 above)
Comment (Gaitholabi) [reply test/Unit/Cache/CacheStrategyConfiguratorTest.php:42]: I noticed that few lines weren't being covered, so this adds some valid data in the loop

All creating new instances, if we are to do more robust assertions we'll have to add some factories which I don't think is worth it

File: src/Model/Ticket/SlaPlan.php (hunk 2 above)
Comment (Gaitholabi) [reply src/Model/Ticket/SlaPlan.php:228]: Should be, it's an array of translations

Did you spot any issues?

**Pull request merged**